bus-mapping/src/circuit_input_builder/input_state_ref.rs

```diff
@@ -1614,7 +1619,6 @@ impl<'a> CircuitInputStateRef<'a> {
                         PrecompileCalls::Sha256
                         | PrecompileCalls::Ripemd160
                         | PrecompileCalls::Blake2F
-                        | PrecompileCalls::ECRecover
```

Comment:
Why do we remove this one here?

Reply:
The list here is for unsupported precompilers and you can see the code below (L1630).

Reply:
Oh, I see now. Sorry for my false comment.